posthog-android/src/main/java/com/posthog/android/internal/PostHogActivityLifecycleCallbackIntegration.kt

```diff
@@ -42,8 +42,8 @@ internal class PostHogActivityLifecycleCallbackIntegration(
     override fun onActivityStarted(activity: Activity) {
         if (config.captureScreenViews) {
             val activityLabel = activity.activityLabel(config)
-            activityLabel?.let {
-                PostHog.screen(it)
+            if (!activityLabel.isNullOrEmpty()) {
+                PostHog.screen(activityLabel)
```

Comment:
I'd be tempted to report "Unlabelled activity"

But then I can imagine the next request would be "How do I turn off reporting unlabelled activities?" so 🤷

Reply:
Yeah, I'd say it's not useful because you won't know which screens they are anyway.